academic_observatory_workflows/workflows/oa_web_workflow.py

```diff
@@ -316,6 +318,32 @@ def clean_ror_id(ror_id: str):
     return ror_id.replace("https://ror.org/", "")
 
 
+def trigger_repository_dispatch(*, token: str, event_type: str, client_payload: Dict = None):
```

Comment:
Could we get rid of the client_payload param? It does not seem to be in use.

Reply:
Yep sure